src/app/tide/spectrum_collection.cc

```diff
@@ -42,6 +42,16 @@ Spectrum::Spectrum(const pb::Spectrum& spec) {
     peak_m_z_.push_back(total / m_z_denom);
     peak_intensity_.push_back(spec.peak_intensity(i) / intensity_denom);
   }
+
+  /// added by Yang
+  ms1_spectrum_number_ = spec.ms1_spectrum_number();
+  iso_window_lower_mz_ = spec.iso_window_lower_mz();
+  iso_window_upper_mz_ = spec.iso_window_upper_mz();
+
+  // carp(CARP_DETAILED_DEBUG, "spectrum_number_:%d \t ms1_spectrum_number_:%d", spectrum_number_, ms1_spectrum_number_);
```

Comment:
do you plan on keeping these carps commented out or not?

Reply:
It's still under development so I will keep it for debugging purpose temporarily